Docs/mc_definition.tex

```diff
@@ -110,13 +110,7 @@ \section{Configuration Management}
 
 Parts are hooked together via connections of their ports, as defined in the connection table.   There are two types of connections:  (1) "hookup" connections and (2) "physical" connections, discussed below.
```

Comment:
Double quotes don't work properly in latex -- you need to use the backtick and single quotes to get them to go the correct direction.

Reply:
Fixed quotes and also added Arduino to drawing.